binary_search_tree/Python/binary_search_tree.py

```diff
@@ -0,0 +1,74 @@
+class Leaf:
+    """
+    Class that represent a leaf node of a binary tree.
+    """
+    

```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpikx6la4g/binary_search_tree/Python/binary_search_tree.py
+++ b/tmp/tmpikx6la4g/binary_search_tree/Python/binary_search_tree.py
@@ -2,7 +2,7 @@
     """
     Class that represent a leaf node of a binary tree.
     """
-    
+
     def __init__(self, value=None, leftChild=None, rightChild=None):
         self.value = value
         self.leftChild = leftChild
```